### ISSUE TITLE
CefSharp using 10g of ram, server 2019

### ISSUE THREAD

**Issue by rklxmok** [canton7/SyncTrayzor#643]
![10gb of ram](https://user-images.githubusercontent.com/67971827/119515894-de95a000-bd3b-11eb-917b-890558d85af0.png)

**Comment by canton7** [comment]
What version of SyncTrayzor?

How long does it take to build to this level? Has it been running for hours/days/weeks/years?

Can you open Chrome, and open `http://localhost:2004`? There should be a link to Syncthing, click it. That opens Chrome dev tools, where you might be able to find something interesting.

**Comment by rklxmok** [comment]
This is 1.1.28 with 1.16.1 and the server instance has been running for 1
day 14 hours, 12 minutes in 2019 hyper v.

Syncing 9 folders from 6 devices

**Comment by canton7** [comment]
Can you try downloading https://cef-builds.spotifycdn.com/cef_binary_90.6.5%2Bg7a604aa%2Bchromium-90.0.4430.93_windows64_client.tar.bz2 and run it with:

```
cefclient.exe --multi-threaded-message-loop --off-screen-rendering-enabled --enable-gpu --no-sandbox --disable-site-isolation-trials
```

Point it at Syncthing's web interface, and see if it takes the same amount of memory after a couple of days?

**Comment by rklxmok** [comment]
Chromium opens port 2004, displays the syncthing instance

**Comment by canton7** [comment]
Yes. Please use the Chrome dev tools to try and find out where all the memory's being used. I'm not too familiar with the Chrome tools I'm afraid, so follow your nose!

**Comment by rklxmok** [comment]
I also went ahead and installed the actual chrome browser, Im sure running
chromium as an executable made some deficiencies of its own. I am
monitoring now.

Thanks for your help
